.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
   - GEM_HOME=vendor/bundle/ruby/2.5.0 bundle install --jobs=3 --retry=3 --deployment --path=${BUNDLE_PATH:-vendor/bundle}
   - GEM_HOME=vendor/bundle/ruby/2.5.0 bundle exec appraisal install
 script:
+  - GEM_HOME=vendor/bundle/ruby/2.5.0 bundle exec rails db:setup
```

Comment:
As you maintain the test schema below, why is this necessary?

Reply:
I think to create the database. Without this line, the build fails but I've not validated whether the maintain schema thing creates db-s or not.